src/module-kount-control/Model/ControlApi/Event.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * Created by Swarming Technology, LLC.
+ * Project: Kount
+ */
+
+namespace Swarming\KountControl\Model\ControlApi;
+
+use Swarming\KountControl\Api\ServiceInterface;
+
+class Event extends AbstractService implements ServiceInterface
+{
+    const ENDPOINT_URI = '/events';
+    const CHALLENGE_DECISION = "Challenge";
+
+    /**
+     * @var array
+     */
+    private $loginResult;
+
+    /**
+     * @inheritdoc
+     */
+    public function preparePayload($sessionId, $clientId)
+    {
+        $userId = $this->customerSession->getCustomerId();
+        // As for now in case "Challenge" decision it always send Success result to Event API
+        if ($this->getLoginResult()['decision'] === self::CHALLENGE_DECISION) {
+            $payload = $this->getSuccessEventPayload($sessionId, $userId, $clientId, $this->getLoginResult());
+        } else {
```

Comment:
No need for else here. if the first condition is met, just `return $this->getSuccessEventPayload($sessionId, $userId, $clientId, $this->getLoginResult());`

Reply:
fixed